docs/content/contribution-guidelines/operations/testing-doc-prs.md

```diff
@@ -1,6 +1,6 @@
 ### Testing a KubeStellar documentation PR
 
-If a contributor has _**not**_ created a sharable preview of a documentation PR [as documented in the documents management overview](/Contribution%20guidelines/operations/document-management/#serving-up-documents-globally-from-a-fork-of-the-repository-via-github) , here are the steps to checkout a git pull request for local testing.
+If a contributor has _**not**_ created a sharable preview of a documentation PR [as documented in the documents management overview](document-management.md#serving-up-documents-globally-from-a-fork-of-the-repository-via-github) , here are the steps to checkout a git pull request for local testing.
```

Comment:
RIght now in docs/content/contribution-guidelines/operations/testing-doc-prs.md I have a version that is working in the render.

Reply:
There are two renders, by GitHub and in the website. Right now, when I look at the GitHub render shown below and follow the link, I get the next GitHub render shown below. This is the best we can do, since GitHub does not understand the inclusion syntax.

<img width="1410" alt="image" src="https://github.com/user-attachments/assets/455db71a-5317-4a2f-a2c9-ebb035ceee1b">

<img width="1430" alt="image" src="https://github.com/user-attachments/assets/21b2deba-3c36-427c-a9b4-54b8c9f61270">

Reply:
Yes. 
We could conceivably add a comment before the include in the markdown that would tell a viewer on github to navigate up to /docs/README.md

Reply:
but you would have to make that comment invisible in the website. You can do that with comment markup, but now it gets ugly. I think it is good enough as is.